lib/constants/enums.ts

```diff
@@ -77,6 +77,11 @@ export enum SwapPhase {
    * error from the swap client.
    */
   SendingPayment = 3,
+  /**
+   * We have completed our outgoing payment and retrieved the preimage which can be used to
+   * settle the incoming payment locked by the same hash.
+   */
+  PreimageResolved = 5,
```

Comment:
Since the enmus are not in line with the swap phases now, maybe add swap phase `1/5` to `5/5` to the comments .

Reply:
Done.